From 05949221e3f996ee36dddfc1b0b8a3eb31b3782e Mon Sep 17 00:00:00 2001
Message-Id: <patch-1.1-05949221e3f-20220319T002709Z-avarab@gmail.com>
In-Reply-To: <cover-0.1-00000000000-20220319T002709Z-avarab@gmail.com>
References: <cover-0.1-00000000000-20220319T002709Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Tue, 15 Mar 2022 12:20:19 +0100
Subject: [PATCH 1/1] Makefile: use '   ', not non-existing $(wspfx_SQ)
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change the use of a non-existing variable added in my
0b6d0bc9246 (Makefiles: add and use wildcard "mkdir -p" template,
2022-03-03) to use the hardcoded whitespace padding for "QUIET" rules
instead. The wspfx_SQ was left from an earlier (rebased out) commit
preceding 0b6d0bc9246[1].

1. https://lore.kernel.org/git/patch-v4-8.9-3733b0c8df1-20220302T124320Z-avarab@gmail.com/

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 shared.mak | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/shared.mak b/shared.mak
index 4e1b62ee99e..50d4596f0d6 100644
--- a/shared.mak
+++ b/shared.mak
@@ -53,7 +53,7 @@ ifndef V
 	QUIET          = @
 	QUIET_GEN      = @echo '   ' GEN $@;
 
-	QUIET_MKDIR_P_PARENT  = @echo $(wspfx_SQ) MKDIR -p $(@D);
+	QUIET_MKDIR_P_PARENT  = @echo '   ' MKDIR -p $(@D);
 
 ## Used in "Makefile"
 	QUIET_CC       = @echo '   ' CC $@;
-- 
2.35.1.1438.g8874c8eeb35

